### DISCUSSION TITLE
Teleport audit events integration with IBM cloud

### DISCUSSION THREAD

**Discussion by PallaviPalisetti** [gravitational/teleport#10965] [Ideas]
Hi there,
For audit events log, available options  for audit events backend are firestore, dynamodb in order to store the audit logs. but, our infrastructure doesn't have either of these. We have IBM as our cloud provider these are available options.
IBM offers a variety of feature-rich, open source NoSQL [databases](https://www.ibm.com/cloud/databases) like [IBM Cloudant](https://www.ibm.com/cloud/cloudant), [Databases for MongoDB](https://www.ibm.com/cloud/databases-for-mongodb), [Databases for Redis](https://www.ibm.com/cloud/databases-for-redis), and [Databases for Elasticsearch](https://www.ibm.com/cloud/databases-for-elasticsearch).

Can we expect these backend to be supported by teleport any time soon.

thanks

**Comment by webvictim** [comment]
Teleport is working on adding support for additional database backends, this will likely be CockroachDB or PostgreSQL though rather than an IBM-native solution.

**Comment by ulysseskan** [comment]
As noted at https://goteleport.com/docs/setup/reference/backends/#postgresqlcockroachdb-preview , Teleport 9.1 provides preview access to configuring PostgreSQL and CockroachDB backends.
